client/src/components/ratingsReviews/RatingsAndReviews.jsx

```diff
@@ -193,6 +206,19 @@ class RatingsAndReviews extends React.Component {
             recommended={this.state.recommended}
             handleRatingProgressFilter={this.handleRatingProgressFilter}
           />
+          {this.state.reviews ?
+            <ReviewsList
+              loadMoreReviews={this.loadMoreReviews}
+              reviews={this.state.reviews}
+              displayedReviews={this.state.displayedReviews}
+              sortOptions={this.sortOptions}
+              handleReviewSort={this.handleReviewSort}
+              getReviews={this.getReviews}
+              currentSort={this.state.sort}
+              product_id={this.state.product_id}
+              count={this.state.count}
+              reviewsLength={this.state.reviewsLength} />
+            : <NewReview />}
```

Comment:
on my widget for the load more, I pass count and all questions to component, and only render question index < count.

Reply:
Got it. I'll consider doing the same for reviews